plugins/Dispersion/Dispersion.cpp

```diff
@@ -0,0 +1,167 @@
+/*
+ * Dispersion.cpp
+ *
+ * Copyright (c) 2023 Lost Robot <r94231/at/gmail/dot/com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#include "Dispersion.h"
+
+#include "embed.h"
+#include "plugin_export.h"
+
+namespace lmms
+{
+
+extern "C"
+{
+
+Plugin::Descriptor PLUGIN_EXPORT dispersion_plugin_descriptor =
+{
+	LMMS_STRINGIFY(PLUGIN_NAME),
+	"Dispersion",
+	QT_TRANSLATE_NOOP("PluginBrowser", "An allpass filter allowing for extremely high orders."),
+	"Lost Robot <r94231/at/gmail/dot/com>",
+	0x0100,
+	Plugin::Effect,
+	new PluginPixmapLoader("logo"),
+	nullptr,
+	nullptr,
```

Comment:
```suggestion
	QT_TRANSLATE_NOOP("PluginBrowser", "An all-pass filter allowing for extremely high orders."),
	"Lost Robot <r94231/at/gmail/dot/com>",
	0x0100,
	Plugin::Effect,
	new PluginPixmapLoader("logo"),
	nullptr,
	nullptr
```
Typo

Reply:
I disagree, "allpass" without the hyphen is also correct and about equally common (same with lowpass and highpass).

Reply:
While I have frequently seen it as "allpass", "all-pass" seems slightly more common and it's what Wikipedia uses. But more importantly, in other places in LMMS, such as the filter tab of instruments and the Dual Filter effect, "all-pass" (and "low-pass"/"high-pass") is used. Some LADSPA plugins use "allpass" but they are 3rd party in origin.